src/webui/www/public/scripts/dynamicTable.js

```diff
@@ -731,6 +731,7 @@ var TorrentsTable = new Class({
         initColumns : function () {
             this.newColumn('priority', '', '#', 30, true);
             this.newColumn('state_icon', 'cursor: default', '', 22, true);
+            this.newColumn('status', '', 'QBT_TR(Status)QBT_TR[CONTEXT=TorrentModel]', 100, true);
```

Comment:
now the status column is the 3rd column counting from the left, I don't think this is an ideal place, move it right (a few lines down in source code).